unity_cogs/verify.py

```diff
@@ -295,6 +296,8 @@ async def make_whois_embed(self, user_data: dict) -> discord.Embed:
         trades = await self.get_trades(user_data['discord']['id'])
         if trades:
             embed.add_field(name='Trades', value=trades, inline=False)
+        usl_status = await usl.fetch_usl_user_data(user_data['reddit']['name'])
```

Comment:
This is always going to return True as long as there is data. This function returns the whole data payload from the API, you should check the 'banned' field within that payload.